### DISCUSSION TITLE
Windows 10 Start Button

### DISCUSSION THREAD

**Discussion by geekinchief2021** [Open-Shell/Open-Shell-Menu#786] [Q&A]
I'm using Open-Shell in Windows 11 and it works great, but I'd really like the Windows 11-style Start button icon. I haven't been able to make one that works (either the transparency is messed up or it doesn't fill the space or both). Does anyone have a Windows 10 Start button icon they've used and can share and is there a reason this can't be built into the app?

**Comment by blackcrack** [comment]
@geekinchief2021 

right mouse key on the startbutton => Startmenu Design =-> 
 - [X] Replace the Startbutton
 - [ ] Replace the Startbutton  <= Uncheck the Checkbox Deep down below, over the Buttons "Aero/Classic/Actual used" !

if this works, can you Close this issue
Best

**Reply by geekinchief2021** [reply]
Sorry, I don't understand. Right clicking on the start button, gives me the option for "settings" and I can upload a custom button, but I can't find that looks like the Windows 10 start button icon and actually works.

**Reply by blackcrack** [reply]
@geekinchief2021 

by the way, The Windows 11 is an Officially upcoming  WinNt, Windows 10 is a actual official used WinNT, both
are belongs Microsoft, if you take or rebuild the Graphics  you put yourself in a legal/illegal/not legal situation.

If we are have there a actually Windows 10 or Windows 11 Startbutton, put us in a legal/illegal/not legally  situation.
This about Windows 10 or 11 Startbutton.. it is simply/just not allowed.. the Design belongs M$

if you want it, take Virtualbox, make a screenshot, take Gimp, make 3 Buttons-pictures vertically and change the transparency..
the Manual over start buttons do you found in the Net.. and google is your friend..

Best
